templates/microcloud/index.html

```diff
@@ -1,405 +1,514 @@
+      <hr class="p-rule" />
+      <div class="col">
+        <div class="p-section--shallow">
+          <h3 class="p-heading--2">Scalable cloud infrastructure</h3>
+        </div>
+      </div>
+      <div class="col">
+        <p>
+          Do you need to keep sensitive workloads in a controlled environment, or are public cloud costs too unpredictable for your business?
+        </p>
+        <p>MicroClouds are an efficient solution for your needs:</p>
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">Resource-efficient to lower costs.</li>
+          <li class="p-list__item is-ticked">Simple to deploy and operate.</li>
+          <li class="p-list__item is-ticked">Easy to scale as your needs grow.</li>
+        </ul>
+        <hr class="p-rule--muted" />
+        <p>
+          <a href="https://ubuntu.com/engage/hypervisor-to-lxd-case-study">Read about LXD's efficiency in this case study&nbsp;&rsaquo;</a>
+        </p>
       </div>
     </div>
-    <div class="col">
-      <p>MicroCloud is lightweight enough that it can run on a developer laptop. You can use it for:</p>
-      <ul class="p-list--divided">
-        <li class="p-list__item is-ticked">Lightweight and constrained disposable testing environments.</li>
-        <li class="p-list__item is-ticked">Simulating or testing complex infrastructure processes.</li>
-        <li class="p-list__item is-ticked">Simulating how your workloads would run in production.</li>
-        <li class="p-list__item is-ticked">Safely experimenting with new technologies.</li>
-      </ul>
-      <hr class="p-rule"/>
-      <p>
-        <a href="https://canonical-microcloud.readthedocs-hosted.com/en/stable/tutorial/">Deploy your own MicroCloud following this tutorial&nbsp;&rsaquo;</a>
-      </p>
+  </section>
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <div class="p-section--shallow">
+          <h3 class="p-heading--2">Efficient dev environment</h3>
+        </div>
+      </div>
+      <div class="col">
+        <p>
+          MicroCloud is lightweight enough that it can run on a developer laptop.
+          <br aria-hidden="true" />
+          You can use it for:
+        </p>
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">Lightweight and constrained disposable testing environments.</li>
+          <li class="p-list__item is-ticked">Simulating or testing complex infrastructure processes.</li>
+          <li class="p-list__item is-ticked">Simulating how your workloads would run in production.</li>
+          <li class="p-list__item is-ticked">Safely experimenting with new technologies.</li>
+        </ul>
+        <hr class="p-rule--muted" />
+        <p>
+          <a href="https://canonical-microcloud.readthedocs-hosted.com/en/stable/tutorial/">Deploy your own MicroCloud following this tutorial&nbsp;&rsaquo;</a>
```

Comment:
For me this link did not open, I got 404. I tagged the stakeholder in the copy doc about this, not a blocker.